tests/test_util.py

```
@@ -1,136 +1,180 @@
+"""Tests for cpp_linter_hooks.util – dynamic PyPI version resolution."""
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)